### ISSUE TITLE
Platinum should be able to kick

### ISSUE THREAD

**Issue by Koshitso** [GMDevinity/FloodIssues#566]
The rule people who break the rules are taking it to another level. As I saw in the forums and ingame, many people complain about staff not being online. I don't argue about that topic, but it's understandable. And therefore, we need to take action against this issue.

What do I expect?
Firstly, I don't expect that the staff will be online 24/7 all day long and try to handle this stuff. As most rule breakers stop violating the rules when an moderator or admin joins, it's pointless anyway imo. So my honest expectation would be, the possibility that Platinum users are from now on able to instantly kick people without voting it up first. You all know how all these votekicks mostly end.

It's up to you guys, if you want people who can do stuff. 

And let me say this, I will not appeal for any position in the team such as moderator / admin or something. I have personal stuff to do and want to play the game and server without moderating 24/7. Giving Platinum these rights to kick people, does not provote them to any kind of moderating force. Therefore it's their will if they want to kick this rule breaking person or not. They don't need to.

Why is this a good idea? I don't need to explain this, really.
Why could this resolate into a bad idea? I mean.. people could easily abuse it, right. But to be honest, if anyone reports this issue or even captures it or something. You can easily revoke the access to platinum or even the power itself.

**Comment by RicksBeehive** [comment]
I agree with your point and there would indeed be a lot of positives to it. I agree that it wouldn't be abused by players, there'd be little incentive for people given that they've put a lot of money forward and it'd be silly to get yourself banned and your rank stripped for abusing.

There's two main issues to this, and why we'll almost definitely never add this.

1) We have a large range of players on the servers, and we are generally very conservative about banning and kicking people. If somebody breaks the rules once non-maliciously then we warn them. Only if they're doing something that needs to be immediately stopped, or they're doing it deliberately do we kick them.
This system is quite a bit different to other servers, and given that some platinum donors may be very young or inexperienced with moderating, the idea of a donator kicking another player over something like saying "Fuck" or "Shit" in chat seems pretty likely. 

2) You make it seem as if being staff is a difficult task, if you're already willing to punish players that are breaking the rules then you're already willing to be a moderator. We have no requirements for playtime etc. or community engagement. Pretty much so long as you demonstrate to us that you're capable of understanding the rules then you'll be accepted as a moderator.

**Comment by Koshitso** [comment]
Hey RicksBeehive. Thanks for your response.

Of course I do understand the first point. You really have a large range of "different" players. Players from all over the world with different minds and different rules they are willing to follow. And I understand that people would definitely kick people for insulting them because I mean it is kinda against the rules.

I know your punishment system well enough that I can say that it works. You don't really need to kick/ban people for doing stuff like once or even twice. Warn them about that and if they are not following the spoken warning, then you can take action against them.

I mean.. being staff is not really difficult, but you first need to take this seriously, that is something where most of the people fail with. The second thing is, that you kinda force yourself to be online as most as possible. And if you're not you could get in trouble or even get demoted.

And yes, I know what you will respond to that. You don't need to be online all the time or even once a day. Every few days would be fine as well. But that's the problem already. If everyone comes online like.. once every week and takes lets say 2 hours moderating, it won't be enough to be honest. The rule breaking people are online in really completely random times. And if a moderator/admin or whatever staff rank joins the server, they have mostly stopped doing it. It's a really difficult topic, I know.

And I will most definitely not apply as a moderating force in this Community. Nor will I do in any community. I had some community already and I know what I'm kinda going into. It's not the community itself, it's kinda me and the things I just said. You are forcing yourself to be online etc etc. And you have a responsibility for the community as well.

And if you are able to kick for example, someone who is definitely not be able to follow easy rules and warnings, as a person who just plays in his Freetime whenever he wants, it would make things kind of easier to be honest. I really understand that it will most likely not be added. But I just wanted to reach out to you guys and talk about this problem.

**Comment by RicksBeehive** [comment]
Alright I'll ignore the stuff about becoming staff as I'm not trying to convince you and if people get certain impressions then that's another issue.

The point of platinum players not really knowing our values etc is the main point.

Just a question, why don't you just use votekick? Or even voteban? Is there not an improvement that could be made to that?

**Comment by Koshitso** [comment]
I'm not 100% sure if the community changed in the meanwhile but the last time I used the votekick feature, no one even bothered to press any button or anything. That's like the most human feedback I can get to be honest. If you're not involved in the situation and getting harassed by another player, why should I care? It's the same thing as it is in DarkRP. There are definitely a few people who vote this seriously. But I can with, with 100% say, that most of the players simply vote F2 which is NO, because it's goon then and they don't want people to be certain things, just in generell.

I might be completely wrong here and you could call me the one, who is spreading the wrong word. But that is something that I have experienced the past.. many years I'm playing Garry's Mod now. Imrpovements to the system? I can give you a so good example how this is working so well. There is a british DarkRP community, where you can even noclip and teleport around in an DarkRP Server. And you have the rights to freeze and even pickup people. No one even thinks about abusing that. And people are loving it.

For example, here are the Platinum VIP benefits when it comes to power " 
You also gain access to the following tools,
these must be used reponsibly and only whilst VOD
Your physgun can pickup players
You can mute players
You can freeze players
You can give yourself godmode
You can jail players
You can set health
You can cloak yourself
You can noclip "

I'm not saying, that we should do anything like that. Not even close. But it's an example where it worked out pretty well. I think you get what I mean.

RicksBeehive. I can really get the point, that platinum players only paid a certain amount of cash. That means not, that they are automatically in the position to be able to moderate correctly. I can only agree with that. And maybe my request is a bit imprecise.  You said you would prefer it, if the rank is given to a trusted person. What is then about that you are being able to idk, apply for a rank, which is available for platinum members? It's basically the same as being stuff, but without being actually stuff.

PS: Sorry for my broken english rn, but I'm really tired.

**Comment by bayrock** [comment]
I think improvements could be made to votekick/voteban if it's not achieving fair results. For example a smaller majority of the votes could result in a kick.

**Comment by Koshitso** [comment]
Hello bayrock.
Thank you for your comment. Yes, some improvements would be life changing. Maybe it's a little too much that a specific donator rank gets the power to kick people. But if you would overhaul the whole system a little, that it's possible with the majority of people to kick the rule breaking person. That would put the system in a whole different place for me and it could be used how it's supposed to be.
